From 6a2fcb296464d1696e4a321697d3a96196a3e8ac Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowi@redhat.com>
Date: Sun, 31 Aug 2025 15:03:59 -0400
Subject: [PATCH] Fix installation of gdb pretty printer

`CMAKE_INSTALL_LIBDIR` may be either a relative or absolute path.
The _FULL_ variables are the safe way to get an absolute path.
---
 src/python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python/CMakeLists.txt b/src/python/CMakeLists.txt
index 56b8fbc6..7325b0a6 100644
--- a/src/python/CMakeLists.txt
+++ b/src/python/CMakeLists.txt
@@ -14,7 +14,7 @@ if(WIN32)
 else()
     set(_version ${PROJECT_VERSION})
     set(_gdb_libname ${_prefix}${_name}${_suffix}.${_version}-gdb.py)
-    set(_gdb_libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
+    set(_gdb_libdir ${CMAKE_INSTALL_FULL_LIBDIR})
     set(_gdb_install_dir ${CMAKE_INSTALL_DATADIR}/gdb/auto-load/${_gdb_libdir})
 endif()
 
-- 
GitLab

